frontend/src/app/testResults/TestResultInput.test.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { render, fireEvent, screen } from "@testing-library/react";
+import { v4 as uuidv4 } from "uuid";
+import TestResultInputForm from "./TestResultInputForm";
+
+jest.mock("uuid");
```

Comment:
We don't need a uuid for `id`, just something unique for a particular run lifetime or specific to a test queue item that has a result. I think the original code got off on the wrong foot with uuid, the only time we should use a function like that is if we're sending something outside the app (for example to the backend). Otherwise a simple counter initialized at React code startup is enough. I added the `useUniqueIds` hook for generating unique ids that way, you could use it here but there are other values as well.

Reply:
I was trying to avoid touching Radio group as much as possible until we have more TS coverage especially if we are moving towards a managed USWDS component library.

`useUniqueIds()` looks so much better then the UUID gen ❤️